leapp/utils/output.py

```diff
@@ -15,35 +18,54 @@ class Color(object):
     yellow = "\033[1;33m" if sys.stdout.isatty() else ""
 
 
-def pretty_block(string, color=Color.bold, width=60):
+def pretty_block_text(string, color=Color.bold, width=60):
     return "\n{color}{separator}\n{text}\n{separator}{reset}\n".format(
         color=color,
         separator="=" * width,
         reset=Color.reset,
         text=string.center(width))
 
 
+@contextmanager
+def pretty_block(text, target, end_text=None, color=Color.bold, width=60):
+    target.write(pretty_block_text(text, color=color, width=width))
+    target.write('\n')
+    yield
+    target.write(pretty_block_text(end_text or 'END OF {}'.format(text), color=color, width=width))
+    target.write('\n')
+
+
 def print_error(error):
     model = ErrorModel.create(json.loads(error['message']['data']))
     sys.stdout.write("{red}{time} [{severity}]{reset} Actor: {actor}\nMessage: {message}\n".format(
         red=Color.red, reset=Color.reset, severity=model.severity.upper(),
         message=model.message, time=model.time, actor=model.actor))
     if model.details:
-        print('Summary: ')
+        print('Summary:')
         details = json.loads(model.details)
         for detail in details:
             print('    {k}: {v}'.format(
                 k=detail.capitalize(),
                 v=details[detail].rstrip().replace('\n', '\n' + ' ' * (6 + len(detail)))))
 
 
+def report_inhibitors(context_id):
+    reports = fetch_upgrade_report_messages(context_id)
+    inhibitors = [report for report in reports if Flags.INHIBITOR in report.get('flags', [])]
+    if inhibitors:
+        text = 'UPGRADE INHIBITED'
+        with pretty_block(text=text, end_text=text, color=Color.red, target=sys.stdout):
+            print('Upgrade has been inhibited due to the following problems:')
+            for position, report in enumerate(inhibitors, start=1):
+                print('{idx:5}. Inhibitor: {title}'.format(idx=position, title=report['title']))
+            print('Consult the pre-upgrade report for details and possible remediation.')
```

Comment:
```suggestion
            print('Consult the pre-upgrade report for details and possible remediation.')
        return True  # if upgrade was inhibited
    return False
```
Return value required for the failure check above.